flink-python/pyflink/table/tests/test_types.py

```diff
@@ -563,6 +563,27 @@ def test_local_zoned_timestamp_type(self):
             ts2 = lztst.from_sql_type(0)
             self.assertEqual(ts.astimezone(), ts2.astimezone())
 
+    def test_timestampLtz_type(self):
```

Comment:
we should reduce code duplication, the same test code exists above. let's remove this change again. if you want to add a test you can simply compare if `DataTypes.TIMESTAMP_LTZ()
 == DataTypes.TIMESTAMP_WITH_LOCAL_TIME_ZONE()
`